opentelemetry-sdk/src/logs/mod.rs

```diff
@@ -212,4 +214,86 @@ mod tests {
             &AnyValue::String("value-from-bag".into())
         ));
     }
+
+    #[test]
+    fn log_suppression() {
+        // Arrange
+        let exporter: InMemoryLogExporter = InMemoryLogExporter::default();
+        let logger_provider = SdkLoggerProvider::builder()
+            .with_simple_exporter(exporter.clone())
+            .build();
+
+        // Act
+        let logger = logger_provider.logger("test-logger");
+        let mut log_record = logger.create_log_record();
+        log_record.set_severity_number(Severity::Error);
+
+        {
+            let _suppressed_context = Context::enter_telemetry_suppressed_scope();
+            logger.emit(log_record);
```

Comment:
nit - add a comment that this emitted log would get suppressed, even if it's obvious. It took time as I was searching for tokio macros to be suppressed :)